index.d.ts

```diff
@@ -300,7 +318,7 @@ declare namespace Moleculer {
 
 		use(...mws: Array<Function>): void;
 
-		findNextActionEndpoint(actionName: string, opts?: GenericObject): string;
+		findNextActionEndpoint(actionName: string, opts?: GenericObject): ActionEndpoint | MoleculerRetryableError;
```

Comment:
Is it not `Errors.MoleculerRetryableError`? In merged 0.13 it gives error: https://travis-ci.org/moleculerjs/moleculer/jobs/392196642#L2390

Reply:
You're right, my test had imported it as an alias which is why it worked. Submitting a PR now